serper.py

```diff
@@ -1,37 +1,52 @@
 import http.client
 import json
 import os
+import requests
+
+GLOBAL_UNUSED_VAR = 42
```

Comment:
The code includes an unused global variable `GLOBAL_UNUSED_VAR`. Removing unused variables improves code clarity and maintainability.